core/src/main/resources/openapi/web3signer-eth2.yaml

```diff
@@ -398,10 +398,10 @@ components:
     AggregateAndProof:
       type: object
       properties:
-        index:
+        aggregator_index:
           type: string
           format: uint64
-        attestation:
+        aggregate:
```

Comment:
changed name, but not the "ref" content - is this expected? (i.e. still have attestation in the $ref

Reply:
Yes that's expected. Well I think it is based on the changes in Teku